djstripe/models.py

```diff
@@ -497,10 +497,11 @@ def sync_current_subscription(self, cu=None):
             return sub_obj
 
     def update_plan_quantity(self, quantity, charge_immediately=False):
+        stripe_subscription = self.stripe_customer.subscription
+        if not stripe_subscription:
+            raise SubscriptionUpdateFailure("Customer does not have a subscription with Stripe")
```

Comment:
Not sure if you missed this comment, but:

What do you think about calling ``sync_subscription`` here to cancel the local subscription if the customer doesn't have one on stripe?